main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-        1) turn on the air conditioner
-        2) adjust the temperature
-        3) activate the defroster
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) radio button
-        2) phone button
-        3) clock
-    Name 3 things you can do with the:   Infotainment System
-        1) turn on the radio
-        2) activate the phone
-        3) tell the time
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) seatbelt
-        2) leather material
-        3) seat adjuster
-    Name 2 things you can do with the:   Seat
-        1) fasten seatbelt
-        2) adjust the seat position
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Gym
+5 properties:
+    1) number of treadmills
+    2) number of barbells
+    3) number of weight benches
+    4) number of protien bars
+    5) stores membership records
+3 things it can do:
+    1) provide excersize equiptment
+    2) provide space to workout
+    3) provide training programs
+ */
+
+/*
+Thing 2) Gas Station
+5 properties:
+    1) stores gas for pumps
+    2) number of gas pumps
+    3) stores product inventory
+    4) number of candy bars
+    5) number of energy drinks
+3 things it can do:
+    1) provide gas to customer
+    2) charge credit card
+    3) provide parking space
+ */
+
+/*
+Thing 3) Coat Rack
+5 properties:
+    1) stores amount of  coats
+    2) stores amount of hats
+    3) number of coats on rack
+    4) number of hats on rack
+    5) number of hooks on rack
+3 things it can do:
+    1) mount to wall
+    2) hang hats
+    3) hang coats
+ */
+
+/*
+Thing 4) Remote Control
+5 properties:
+    1) on/off switch
+    2) contains batteries
+    3) left/right buttons
+    4) up/down buttons
+    5) sits on table
```

Comment:
`sits on table` is an action, not a property.  `location` is a property, which can be represented as a `std::string`
- [ ] revise